proto/internalapi/sensor/collector_iservice.proto

```diff
@@ -3,16 +3,22 @@ syntax = "proto3";
 package sensor;
 
 import "google/protobuf/empty.proto";
-import "internalapi/common/collector_runtime_config.proto";
+import "storage/cluster.proto";
 
 option go_package = "./internalapi/sensor;sensor";
 
 message MsgToCollector {
   oneof msg {
-    common.CollectorRuntimeConfig config_with_cluster = 1;
+    storage.CollectorConfig collector_config = 1;
+  }
+}
+
+message MsgFromCollector {
+  oneof msg {
+    storage.CollectorConfig collector_config = 1;
   }
 }
 
 service CollectorService {
-  rpc Communicate(google.protobuf.Empty) returns (stream MsgToCollector);
+  rpc Communicate(stream MsgFromCollector) returns (stream MsgToCollector);
```

Comment:
It turned out that this didn't work if it was assumed that collector doesn't return anything. This is also the reason for `message MsgFromCollector`.